frame_hdr_events.txt

```diff
@@ -37,13 +38,25 @@ The return value will be formatted like this:
             id: hashed input
             descriptor_name: string
             event_type_id: integer
-            run_header_id: hashed input
-            tag: list
-            type_descriptor: ??
-            data_keys: {
-                pv1_alias: {pv1_name: 'XF:23ID1-OP{Slt:3-Ax:X}Mtr'},
-                pv2_alias: {pv2_name: 'pv1_name: 'XF:23ID1-OP{Slt:3-Ax:X}Mtr'},
-                ....
+            run_header_id: sha1hash
+            type_descriptor: string
+            data_keys: [
+                pv1,
+                ...
+                pvN
+            },
+            external_data_keys: {
+                ext_pv1,
+                ...
+                ext_pvM
```

Comment:
I recall we said that `data_keys` would be all-inclusive, so for illustrative purposes the list of `data_keys` should also include `ext_pv1`, `ext_pv2`, etc. "Internal" data keys can be computed as needed using set logic.